include/sound/sof/xman.h

```diff
@@ -46,9 +46,27 @@ struct xman_elem_header {
 
 /* Extended manifest elements identificators */
 enum xman_elem_type {
+	XMAN_ELEM_FW_VERSION	= 1,
 	XMAN_ELEM_CC_VERSION	= 3,
 }
 
+/* FW version */
+struct xman_fw_version {
+	struct xman_elem_header hdr;
+	uint16_t major;
+	uint16_t minor;
+	uint16_t micro;
+	uint16_t build;
+	uint8_t date[12];
+	uint8_t time[10];
+	uint8_t tag[6];
+	uint32_t abi_version;
+	uint32_t flags;
+
+	/* reserved for future use */
+	uint32_t reserved[4];
```

Comment:
only 4?  Just sand-bag and plan for extensions

Reply:
Copied from existing code

Reply:
what existing code? Just sand bag and add e..g 16 words. Who cares about size here?